packages/tpetra/core/src/Tpetra_Export_def.hpp

```diff
@@ -517,6 +524,13 @@ namespace Tpetra {
     host_remote_lids_type remoteLIDs
       (view_alloc_no_init ("remoteLIDs"), numRemoteIDs);
 
+    // FENCE REVIEW - CONFIRMED FAILURE
+    //   Testing: This code is exercised by unit tests.
+    //   GTX960:  Fails with CUDA_LAUNCH_BLOCKING=0 and fence removed. TpetraCore_SubmapExportTests_MPI_2
+    //   White:   Not checked since failure already confirmed on GTX960.
+    //   Plan:    Potentially refactor UVM access below and remove fence.
+    //   Notes:
+
```

Comment:
In setupRemote(): It isn't clear why a fence needs to be right here.  MPI gets called before this (on a host array, so that probably doesn't need fencing.